### ISSUE TITLE
wrong-type-argument window-live-p #<window 69>

### ISSUE THREAD

**Issue by brianmcgillion** [hlissner/doom-emacs#4133]
**What did you expect to happen?**

Using org-roam - opening the `org-roam` buffer to check backlinks. Should be presented with a window and buffer that lists the backlinks. instead the error `wrong-type-argument window-live-p #<window 69>` is displayed.

Closing the org-roam buffer and opening it again does not show the error, however, the backlinks buffer (`org-roam`) is a blank empty buffer.

**What actually happened?**
with the doom-debug-mode enabled the following error is shown in place of the backlink buffer

**Additional details:**
<details><pre>
Explain-pause-mode: please report this bug by creating a Github
issue at https://github.com/lastquestion/explain-pause-mode. Explain-pause-mode
is now _disabled_ so you can continue to hopefully use Emacs. Info follows:

explain-pause version: 0.1
emacs version: 27.1

not top level in wrap-native for #<subr recursive-edit>
current
#s(explain-pause-command-record root-emacs nil nil 25162 (24465 43147 396540 484000) nil nil nil nil 0)

Backtrace:
  explain-pause-report-measuring-bug("not top level in wrap-native for #<subr recursive-..." "current" #s(explain-pause-command-record :command root-emacs :native nil :parent nil :executing-time 25162 :entry-snap (24465 43147 396540 484000) :too-slow nil :is-profiled nil :under-profile nil :profile nil :depth 0))
  explain-pause--wrap-native(#<subr recursive-edit>)
  apply(explain-pause--wrap-native #<subr recursive-edit> nil)
  recursive-edit()
  debug(error (wrong-type-argument window-live-p #<window 69>))
  redisplay_internal\ \(C\ function\)()
</pre></details>

**Steps to reproduce:**
1. Open an org-roam file
2. open the backlinks buffer (assumes that it does not automatically show)
3. error happens

**System information:**
<details><pre>

```
SYSTEM  type       gnu/linux
	config     x86_64-pc-linux-gnu
	shell      /bin/bash
	uname      Linux 5.4.0-52-generic #57-Ubuntu SMP Thu Oct 15 10:57:00 UTC 2020 x86_64
	path       (~/.pyenv/shims ~/.pyenv/bin ~/.local/bin ~/bin /usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin /usr/games /usr/local/games /snap/bin ~/.emacs.d/bin /snap/bin /var/lib/snapd/snap/bin /usr/lib/emacs/27.1/x86_64-linux-gnu)
EMACS   dir        ~/.emacs.d/
	version    27.1
	build      Sep 19, 2020
	buildopts  --build=x86_64-linux-gnu --prefix=/usr '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' --sysconfdir=/etc --localstatedir=/var --disable-silent-rules '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' --disable-maintainer-mode --disable-dependency-tracking --prefix=/usr --sharedstatedir=/var/lib --libexecdir=/usr/lib --localstatedir=/var/lib --infodir=/usr/share/info --mandir=/usr/share/man --enable-locallisppath=/etc/emacs:/usr/local/share/emacs/27.1/site-lisp:/usr/local/share/emacs/site-lisp:/usr/share/emacs/27.1/site-lisp:/usr/share/emacs/site-lisp --program-suffix=27 --with-modules --with-file-notification=inotify --with-mailutils --with-harfbuzz --with-json --with-x=yes --with-x-toolkit=gtk3 --with-lcms2 --with-cairo --with-xpm=yes --with-gif=yes --with-gnutls=yes --with-jpeg=yes --with-png=yes --with-tiff=yes --with-xwidgets 'CFLAGS=-g -O2 -fdebug-prefix-map=/build/emacs27-bifpWT/emacs27-27.1~1.git86d8d76aa3=. -fstack-protector-strong -Wformat -Werror=format-security -no-pie' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro -no-pie'
	features   XPM JPEG TIFF GIF PNG RSVG CAIRO SOUND GPM DBUS GSETTINGS GLIB NOTIFY INOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE HARFBUZZ M17N_FLT LIBOTF ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD JSON PDUMPER LCMS2 GMP
	traits     (batch server-running envvar-file)
DOOM    dir        ~/.doom.d/
	version    2.0.9
	build      HEAD -> bibtex c10a09bc0 2020-10-22 09:35:31 +0400
	elc-files  141
	modules    (:completion (company +childframe) (helm +fuzzy) :ui doom doom-dashboard hl-todo hydra modeline nav-flash ophints (popup +defaults) treemacs unicode vc-gutter (window-select +numbers) workspaces zen :editor file-templates fold (format +onsave) snippets word-wrap :emacs (dired +icons) electric (ibuffer +icons) (undo +tree) vc :term eshell vterm :checkers (syntax +childframe) (spell +aspell +everywhere) grammar :tools biblio (debugger +lsp) docker editorconfig (eval +overlay) (lookup +dictionary +offline +wordnet +docsets) (lsp +peek) magit make pass pdf taskrunner :lang (cc +lsp) data emacs-lisp (go +lsp) latex markdown (org +dragndrop +pandoc +present +gnuplot +roam +noter) (python +lsp +pyenv) (sh +lsp) web :app (rss +org) :config literate (default +bindings +smartparens))
	packages   ((elfeed-goodies) (org-mode :recipe (:repo https://code.orgmode.org/bzg/org-mode.git :files (*.el lisp/*.el contrib/lisp/*.el)) :pin 8402c4a778) (org-roam-server) (company-tabnine) (super-save) (crux) (imenu-anywhere) (atomic-chrome))
	unpin      (org-roam helm)
	elpa       (n/a)
```

</pre></details>

**Comment by hlissner** [comment]
I cannot reproduce this on the pinned version of org-roam. Please repin org-roam and let me know if that fixes the issue.

**Comment by brianmcgillion** [comment]
sorry I had been testing combinations to see what might fix it. But the pinned and unpinned were the same after your recent bump, so I left the unpinned.

What happened to be the issue is that I ended up with a lot of cyclical links, I recently changed to using org-id and the combination of org-roam + org-id in some cases lead to these cycles. I will raise an issue with org-roam that they could notify, or maybe even identify if it can not create the backlinks because of this recursion.
